### ISSUE TITLE
ReuseFolderStructure flag works but throws an error because of an issue in Join-AdminUNC

### ISSUE THREAD

**Issue by hintbw** [sqlcollaborative/dbatools#666]
### Is this a *feature* OR *bug*:
*Bug*


### System Details

- Operating system name and version: Win 8.1
- Output from `$PSVersionTable`:

```
Evaluate $PSVersionTable in PowerShell and paste the output here
```
Name                           Value                                                                                                                                                                                                                            
----                           -----                                                                                                                                                                                                                            
PSVersion                      5.1.14409.1005                                                                                                                                                                                                                    
PSEdition                      Desktop                                                                                                                                                                                                                          
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                                                                                                                          
BuildVersion                   10.0.14409.1005                                                                                                                                                                                                                  
CLRVersion                     4.0.30319.42000                                                                                                                                                                                                                  
WSManStackVersion              3.0                                                                                                                                                                                                                              
PSRemotingProtocolVersion      2.3                                                                                                                                                                                                                              
SerializationVersion           1.1.0.1
- Output of dbatools version:
0.8.702

```
Evaluate (Get-Module dbatools -ListAvailable).Version and paste output here
```



- SQL Server version for source/target
2012

### Steps to Reproduce
Use -ReuseFolderStructure flag on a Copy-SQLDatabase function call


### Action Results
Uses ReuseFolderStructure works but throws an error when used. It looks like it is just mising a dot filename after referencing the zero index item of the array that is returned.


### Expected Results
ReuseFolderStructure will find the path of the files and be utilized correctly and not throw an error.

### Attached Logs or any exception errors

- Use of Start-Transcript can help collection of console output and exceptions
- Certain commands will generate an exception log that you can find in following directory: `dir $env:USERPROFILE\Documents\dbatools-exceptions.txt`

**Comment by niphlod** [comment]
closing as its a duplicate of #115

**Comment by potatoqualitee** [comment]
Should be fixed now D: in 0.9.127
